### PR TITLE
Upgrade to Chromium 63

### PR DESCRIPTION
**Pull request by alexeykuzmin**
libcc's PR: https://github.com/electron/libchromiumcontent/pull/407
Project board: https://github.com/electron/electron/projects/7

### CODE REVIEW

**Comment by jarek-foksa** [comment]
Is there going to be any release based on Chromium 61 or 62? Or are you planning to skip them and release Electron 2.0 when Chromium 63 upgrade is finished?

**Comment by alexeykuzmin** [comment]
> Is there going to be any release based on Chromium 61 or 62?

@jarek-foksa Probably. Maybe. No promises. Stay tuned.

**Comment by bbert** [comment]
Hi,
Will you also provide afterwards a version with chrome 64?
Since chrome 64 will provide license persistency, that will be worthy to have it in Electron.

**Comment by alexeykuzmin** [comment]
> Will you also provide afterwards a version with chrome 64?

Hi, @bbert 
We will definitely provide a version of Electron with Chromium 64 or newer in the future.

**Comment by setpixel** [comment]
Hi guys 👍 

I know the answer is: when it's done.

But do you have a very rough goal of when there will be a release with a chromium beyond 59? 💃 

❤️ ❤️ ❤️ ❤️ ❤️ ❤️

**Comment by alexeykuzmin** [comment]
@setpixel Soon.

**Comment by wesamco** [comment]
hoping it's a Chromium version >= 64.0.3240.0, so puppeteer (which uses DevTools Protocol) can work with it.
https://github.com/electron/spectron/issues/240

**Comment by alexeykuzmin** [comment]
@wesamco Sorry, but the next release will probably have Chromium 61.

**Comment by taha** [comment]
@alexeykuzmin I'm quite confused. I'm working on a private folk of Electron at the moment. Back then when I was starting I thought it would be a good idea to folk it from 1-8-x. It had Chromium 61. Now I rebased 1.8.2-beta.5, and it uses Chromium 59. Will it revert back to 61 before the actual release of 1.8.2?

**Comment by alexeykuzmin** [comment]
@taha `1-8-x` branch has never had Chromium 61.
You probably had the `vendor/libchromiumcontent` submodule out of sync.

**Comment by petrbrzek** [comment]
@alexeykuzmin any ETA for this? Also what about #11230, is it going to be closed?

**Comment by alexeykuzmin** [comment]
@petrbrzek No ETA. Chromium 62 is going to be merged and released before Chromium 63.

**Comment by zcbenz** [comment]
I got a rather weird linker error on Linux x64:

```
/usr/bin/ld: /home/builduser/project/vendor/download/libchromiumcontent/shared_library/libmirclient.so.9: __cxa_finalize: invalid needed version 2
/home/builduser/project/vendor/download/libchromiumcontent/shared_library/libmirclient.so.9: error adding symbols: Bad value
```

**Comment by alexeykuzmin** [comment]
@deepak1556 Check this out https://github.com/nodejs/node/pull/16271

**Comment by zcbenz** [comment]
Encountered a strange linking error on Windows:

```
pdfium.lib(cpdfsdk_annotiteration.obj) : error LNK2019: unresolved external symbol __std_reverse_trivially_swappable_8 referenced in function "void __cdecl std::_Reverse_unchecked1<class CPDFSDK_Annot * *>(class CPDFSDK_Annot * * const,class CPDFSDK_Annot * * const,struct std::integral_constant<unsigned __int64,8>)" (??$_Reverse_unchecked1@PEAPEAVCPDFSDK_Annot@@@std@@YAXQEAPEAVCPDFSDK_Annot@@0U?$integral_constant@_K$07@0@@Z)
fpdfapi.lib(cpdf_psengine.obj) : error LNK2019: unresolved external symbol __std_reverse_trivially_swappable_4 referenced in function "void __cdecl std::_Reverse_unchecked1<float *>(float * const,float * const,struct std::integral_constant<unsigned __int64,4>)" (??$_Reverse_unchecked1@PEAM@std@@YAXQEAM0U?$integral_constant@_K$03@0@@Z)
```

Found some information at https://developercommunity.visualstudio.com/content/problem/205947/binary-compatibility-possibly-broken-using-stdreve.html but still not sure how to fix it.

**Comment by zcbenz** [comment]
Updated VS 2017 on my machine and fixed the error, it seems that VS 2017 has binary compatibility issues between its minor releases.

**Comment by alexeykuzmin** [comment]
@gerhardberger Hi! If you have time can you please take a look at compilation errors in this branch with OSR enabled? (It's disabled in features.gypi). Thanks!

**Comment by gerhardberger** [comment]
@alexeykuzmin yeah, for sure, will take a look at it later in the week.

File: atom/common/api/features.cc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  return false;
+#endif
+}
+
+void Initialize(v8::Local<v8::Object> exports,
+                v8::Local<v8::Value> unused,
+                v8::Local<v8::Context> context,
+                void* priv) {
+  mate::Dictionary dict(context->GetIsolate(), exports);
+  dict.SetMethod("isPDFViewerEnabled", &IsPDFViewerEnabled);
```
Comment (poiru) [inline atom/common/api/features.cc]: Can we make this a property?

**Comment by alexeykuzmin** [comment]
@electron/printing @electron/docs @electron/notifications 
please review the PR

File: atom/common/api/features.cc (hunk 1 above)
Comment (alexeykuzmin) [reply atom/common/api/features.cc]: @poiru Maybe ) Thanks!

**Pull request merged**